tailwind.config.mjs

```diff
@@ -2,7 +2,134 @@
 export default {
   content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'josh-bg': 'hsl(210deg, 30%, 8%)',
+        'josh-text': 'hsl(0deg, 0%, 100%)',
+        'josh-primary': 'hsl(333deg, 100%, 52%)', // Pink
+        'josh-secondary': 'hsl(180deg, 100%, 40%)', // Teal
+        'josh-accent': 'hsl(250deg, 100%, 70%)', // Purple
+        'josh-gray': 'hsl(210deg, 15%, 60%)',
```

Comment:
Again way too on the nose about the copying here 

We don't need the literal color style from it and caffeine and embedding it into the properties is even worth